src/components/ChosenCountry/ChosenCountryMobile/ChosenCountryMobile.test.js

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable testing-library/no-node-access */
+/* eslint-disable testing-library/no-container */
+import React from 'react';
+import { MockedProvider } from '@apollo/client/testing';
+import { render } from '@testing-library/react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { ChosenCountryMobile } from './ChosenCountryMobile';
+
+const mockData = {
+  country: {
+    capital: 'Andorra la Vella',
+    code: 'AD',
+    continent: { name: 'Europe', __typename: 'Continent' },
+    currency: 'EUR,UA,RUS',
+    languages: [
+      { name: 'Catalan' },
+    ],
+    name: 'Catalan',
+    phone: '376',
+  },
+};
+
+describe('Test ChosenCountryDesktop', () => {
+  it('should render ChosenCountryMobile element by class "chosen-country-mob"', () => {
+    const { container } = render(
+      <MockedProvider>
+        <Router>
+          <ChosenCountryMobile data={mockData} />
+        </Router>
+      </MockedProvider>
+    );
+    expect(container.getElementsByClassName('chosen-country-mob')[0]).toBeInTheDocument();
+  });
+
+  it('should render ChosenCountryMobile element by class "chosen-country-mob__block"', () => {
+    const { container } = render(
+      <MockedProvider>
+        <Router>
+          <ChosenCountryMobile data={mockData} />
+        </Router>
+      </MockedProvider>
+    );
+    expect(container.getElementsByClassName('chosen-country-mob__block')[0]).toBeInTheDocument();
+  });
+
+  it('should render ChosenCountryMobile element by class "chosen-country-mob__content"', () => {
+    const { container } = render(
+      <MockedProvider>
+        <Router>
+          <ChosenCountryMobile data={mockData} />
+        </Router>
+      </MockedProvider>
+    );
+    expect(container.getElementsByClassName('chosen-country-mob__content')[0]).toBeInTheDocument();
+  });
+
+  it('should render ChosenCountryMobile element by class "chosen-country-mob__content-data"', () => {
+    const { container } = render(
+      <MockedProvider>
+        <Router>
+          <ChosenCountryMobile data={mockData} />
+        </Router>
+      </MockedProvider>
+    );
+    expect(container.getElementsByClassName('chosen-country-mob__content-data')[0]).toBeInTheDocument();
+  });
+
+  it('should render ChosenCountryMobile element by class "chosen-country-mob__name"', () => {
+    const { container } = render(
+      <MockedProvider>
+        <Router>
+          <ChosenCountryMobile data={mockData} />
+        </Router>
+      </MockedProvider>
+    );
+    expect(container.getElementsByClassName('chosen-country-mob__currency')[0]).toBeInTheDocument();

```

Comment:
This and next tests doing basically the same - they check whether element with class `chosen-country-mob__currency` is present in DOM. Is there any mistake here?

Reply:
Also, you can do all these checks in single test. No need to create a separate tests if inputs and states of the component are not changed.